src/utils/installer.js

```
@@ -64,8 +88,10 @@ async function promptInstallation(toolName) {
     console.log(chalk.gray('\nYou can install it manually later:'));
     if (toolName.includes('Claude')) {
       console.log(chalk.white('  npm install -g @anthropic-ai/claude-code'));
-    } else {
+    } else if (toolName.includes('Claude')){
```

Biome (2.1.2)

[error] This branch can never execute. Its condition is a duplicate or covered by previous conditions in the if-else-if chain.



(lint/suspicious/noDuplicateElseIf)